l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -41,14 +42,15 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
 
     /// @notice StateTransitionManagers that are registered, and ZKchains that use these STMs can use this bridgehub as settlement layer.
     mapping(address _stateTransitionManager => bool) public stateTransitionManagerIsRegistered;
+
     /// @notice we store registered tokens (for arbitrary base token)
-    mapping(address _baseToken => bool) public tokenIsRegistered;
+    mapping(address _baseToken => bool) public __DEPRECATED_tokenIsRegistered;
```

Comment:
You said its not final, but just writing it down: assetIdIsRegistered needs to be added.